aspnetcore/fundamentals/dependency-injection.md

```diff
@@ -628,7 +628,7 @@ The debug console shows the following output after each refresh of the Index pag
 ```console
 Service1: IndexModel.OnGet
 Service2: IndexModel.OnGet
-Service3: IndexModel.OnGet
+Service3: IndexModel.OnGet, MyKey = {MyKey}  //The "MyKey" is according to your config
```

Comment:
Let's keep this code, run it, save the output and fix up the output.
Use 

```
{
  "MyKey": "MyKey from appsettings.Developement.json",
  "Logging": {
    "LogLevel": {
      "Default": "Information",
      "System": "Debug",
      "Microsoft": "Error"
    }
  }
}
```

Reply:
Ok, and I change it. Also I find that there is a same mistake in .net 3.x docs and fix it, too.